elastic/apm-server
# Translate more OpenTelemetry resource conventions

## Motivation/summary

Translate more OpenTelemetry resource conventions, focusing on common conventions which already have a clear mapping to our data model. Add fields to metadata model types for missing ECS fields.

## Checklist


## How to test these changes

1. Run an OpenTelemetry-instrumented application with rich instrumentation (e.g. Java auto-instrumentation), pointed at apm-server
2. Wait for events to be recorded, check:
   - Labels for untranslated attributes
   - Captured field values are sensible

## Related issues

https://github.com/elastic/apm-server/issues/4714
